homeassistant/components/lupusec/config_flow.py

```diff
@@ -0,0 +1,89 @@
+""""Config flow for Lupusec integration."""
+
+import ipaddress
+import logging
+import socket
+
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_HOST, CONF_NAME, CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_HOST): str,
+        vol.Required(CONF_USERNAME): str,
+        vol.Required(CONF_PASSWORD): str,
+        vol.Optional(CONF_NAME): str,
+    }
+)
+
+
+class LupusecConfigFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
```

Comment:
General question: can we get any kind of identifier from the devices? Some kind of serial number or something along those lines?

Reply:
Added device connection check

Reply:
I mean from a device. As in, what am I connecting with (I don't know what a Lupusec is). Can we get an unique identifier for this config entry? Some kind of account Id, serial number for a hub, mac address of a hub, something along those lines.